saleor/graphql/page/bulk_mutations.py

```diff
@@ -19,6 +23,22 @@ class Meta:
         error_type_class = PageError
         error_type_field = "page_errors"
 
+    @classmethod
+    def perform_mutation(cls, _root, info, ids, **data):
```

Comment:
IMO we should add `traced_atomic_transaction` here. We do not want the situation where attribute values are deleted but pages are not.